### DISCUSSION TITLE
Adding contact resistance to P2D model

### DISCUSSION THREAD

**Discussion by manjunathnilugal** [pybamm-team/PyBaMM#2137] [Q&A] [answered]
Dear All,

I need to include the contact resistance as shown in attached figure in the source code. I need to change the terminal voltage equation in the source code. I am not able to figure it out in the lithium ion full battery **base dfn model**. Kindly help me to resolve this issue.
**It is an Ohmic resistance which includes combined contact resistance of current collector, electrodes contact resistance, and electrolyte resistance.**
If there are any similar documents, kindly share with me know.
Anode := 25 ohm.cm2
Cathode := 65 ohm.cm2

<img width="272" alt="image" src="https://user-images.githubusercontent.com/82285213/176701082-86b6607f-92e0-4fbd-b68b-21d09cdeb4da.png">

**Comment by brosaplanella** [comment, marked as answer]
This seems to be answered already in #2116
